examples/js-sub.c

```diff
@@ -130,21 +169,47 @@ int main(int argc, char **argv)
 
     if (s == NATS_OK)
     {
-        if (pull)
+        if (pull && async)
+        {
+            jsOpts.PullSubscribeAsync.FetchSize = 17;
+            jsOpts.PullSubscribeAsync.KeepAhead = 7;
+            jsOpts.PullSubscribeAsync.CompleteHandler = _completeFetchCb;
+            jsFetchRequest lifetime = {
```

Comment:
Apparently, on my Windows VM, this type of initialization causes warning "C4204: nonstandard extension used: non-constant aggregate initializer".
One way to solve would be:
```
jsFetchRequest lifetime;

jsFetchRequest_Init(&lifetime);
lifetime.Batch   = (int) total;
lifetime.NoWait = true;
lifetime.Expires = 1 * HOUR_NANO;
```

Reply:
Interesting why are we not getting it in CI with a /W4 (same [c]makefile anyway). Different compiler versions? I'll change and try to stay away from the pattern.

Reply:
It may be my (very old) Windows VM. Talking about `/W4`, when I compile on Windows, I get a warning, for each file compiled, saying "overriding `/W3` with `/W4`.

But again, I would not be surprised that it is due to my old Windows VM. If there is no complaint in the Windows build we have on GA, maybe ignore my comment?

Reply:
I changed it.

In general, I would love to turn on warnings == errors on all platforms, but there are a few warnings (including disabling /W4 in test.c and now on your VM) that would get in the way